package.json

```
@@ -10,7 +10,8 @@
 		"lint": "eslint *.js src/ lib/ test/",
 		"lint-fix": "eslint --fix *.js src/ lib/ test/",
 		"eslint-check": "eslint --print-config . | eslint-config-prettier-check",
-		"test": "npm run lint && jest"
+		"test": "npm run lint && jest",
```

You could just use `npm run test` for testing, so that you don't need to change 2 locations if you change how tests are run.

```suggestion
		"test": "npm run lint && npm run test",
```